docs/request.md

```diff
@@ -44,8 +44,9 @@ Argument                  | Type                              | Required | Descr
 ------------------------- | --------------------------------- | -------- | ---
 `options`                 | `Object`                          | Yes      | The request options to pass.
 `options.method`          | `String`                          | No       | The HTTP method to use. This value should be one of the following: `GET`, `POST`, `PUT`, `PATCH`, `DELETE`, `HEAD` or `OPTIONS`. Defaults to `GET`.
-`options.url`             | `String`                          | Yes      | The [path name](paths.md) to send the request to, optionally interpolated with values from `options.data`.
-`options.data`            | `any`                             | No       | The data to be interpolated into the URL and serialized into the querystring (for GET requests) or body (for other types of requests).
+`options.url`             | `String`                          | Yes      | The [path name](paths.md) to send the request to, optionally interpolated with values from `options.params`.
+`options.params`            | `any`                           | No       | The data to be interpolated into the URL and serialized into the querystring (for GET requests).
```

Comment:
Pretty sure this should be `Object` instead of `any`...?

Reply:
Changed